### ISSUE TITLE
sdb2 error on exit: could not marshal model to XML

### ISSUE THREAD

**Issue by crosscodr** [mathisdt/sdb2#64]
The version of commit b182d74a33de80fd19f4e3959e8e87e1749e677d throws an exception on exit:

```
could not marshal model to XML

java.lang.IllegalStateException: could not marshal model to XML
    org.zephyrsoft.sdb2.model.XMLConverter.fromPersistableToXML(XMLConverter.java:41)
    org.zephyrsoft.sdb2.MainController.saveSettings(MainController.java:514)
    org.zephyrsoft.sdb2.MainController.saveAll(MainController.java:365)
    org.zephyrsoft.sdb2.MainController.prepareClose(MainController.java:360)
CAUSED BY:
javax.xml.bind.MarshalException: null
    com.sun.xml.bind.v2.runtime.MarshallerImpl.write(MarshallerImpl.java:331)
    com.sun.xml.bind.v2.runtime.MarshallerImpl.marshal(MarshallerImpl.java:256)
    javax.xml.bind.helpers.AbstractMarshallerImpl.marshal(AbstractMarshallerImpl.java:110)
    org.zephyrsoft.sdb2.model.XMLConverter.fromPersistableToXML(XMLConverter.java:39)
CAUSED BY:
com.sun.istack.SAXException2: com.sun.xml.bind.api.AccessorException: java.lang.IllegalStateException: found no marshaller for setting value type "class [C"
    com.sun.xml.bind.v2.runtime.XMLSerializer.reportError(XMLSerializer.java:247)
    com.sun.xml.bind.v2.runtime.XMLSerializer.reportError(XMLSerializer.java:262)
    com.sun.xml.bind.v2.runtime.ClassBeanInfoImpl.serializeBody(ClassBeanInfoImpl.java:370)
    com.sun.xml.bind.v2.runtime.XMLSerializer.childAsXsiType(XMLSerializer.java:696)
CAUSED BY:
com.sun.xml.bind.api.AccessorException: java.lang.IllegalStateException: found no marshaller for setting value type "class [C"
    com.sun.xml.bind.v2.runtime.reflect.AdaptedAccessor.get(AdaptedAccessor.java:78)
    com.sun.xml.bind.v2.runtime.property.SingleElementNodeProperty.serializeBody(SingleElementNodeProperty.java:133)
    com.sun.xml.bind.v2.runtime.ClassBeanInfoImpl.serializeBody(ClassBeanInfoImpl.java:360)
    com.sun.xml.bind.v2.runtime.XMLSerializer.childAsXsiType(XMLSerializer.java:696)
CAUSED BY:
java.lang.IllegalStateException: found no marshaller for setting value type "class [C"
    org.zephyrsoft.sdb2.util.converter.SettingValueAdapter.marshal(SettingValueAdapter.java:218)
    org.zephyrsoft.sdb2.util.converter.SettingValueAdapter.marshal(SettingValueAdapter.java:41)
    com.sun.xml.bind.v2.runtime.reflect.AdaptedAccessor.get(AdaptedAccessor.java:76)
    com.sun.xml.bind.v2.runtime.property.SingleElementNodeProperty.serializeBody(SingleElementNodeProperty.java:133)
```

Environment:
Windows 10.

**Comment by mathisdt** [comment]
I'm sorry to hear that. Could you please post your database file if that's legally possible or else [mail it to me](mailto:sdb2-github@muchmail.de) so I can debug this?

**Comment by crosscodr** [comment]
I send you a pm.

**Comment by mathisdt** [comment]
This is fixed with commit f9d4b0b.
